tests/test_loaders.py

```diff
@@ -1,22 +1,34 @@
-from typing import Any, Dict
+from typing import Any, Dict, List
 
 import pytest
 from numpy import float64
-from pandas import DataFrame
+from pandas import DataFrame, concat, read_csv
+from pandas.api.types import is_float_dtype
 
 from gnatss.configs.main import Configuration
-from gnatss.constants import SP_DEPTH, SP_SOUND_SPEED
-from gnatss.loaders import load_configuration, load_sound_speed
+from gnatss.constants import SP_DEPTH, SP_SOUND_SPEED, TT_DATE, TT_TIME
+from gnatss.loaders import load_configuration, load_sound_speed, load_travel_times
 from gnatss.main import gather_files
 from tests import TEST_DATA_FOLDER
 
+PARSED_FILE = "parsed"
+
 
 @pytest.fixture
 def all_files_dict() -> Dict[str, Any]:
     config = load_configuration(TEST_DATA_FOLDER / "config.yaml")
     return gather_files(config)
 
 
+@pytest.fixture
+def all_files_dict_j2k_travel_times() -> Dict[str, Any]:
+    config = load_configuration(TEST_DATA_FOLDER / "config.yaml")
+    config.solver.input_files.travel_times.path = (
+        "./tests/data/2022/NCL1/**/WG_*/pxp_tt_j2k"
+    )

```

Comment:
Seems like `load_configuration` is called multiple times in this whole test module. I think it would be better to extract this out to a fixture like: 

This configuration fixture can even live in `conftest.py` so it can be used to any other test fixture or test methods that uses this object.

```python
@pytest.fixture
def configuration() -> Configuration:
    return load_configuration(TEST_DATA_FOLDER / "config.yaml")
```

For the specific test fixture above, it might be best to make a configuration copy to ensure that you're not modifying the original object once you make a singular configuration fixture:

```python
@pytest.fixture
def all_files_dict_j2k_travel_times(configuration) -> Dict[str, Any]:
    config = configuration.model_copy(deep=True)
    config.solver.input_files.travel_times.path = (
        "./tests/data/2022/NCL1/**/WG_*/pxp_tt_j2k"
    )
    ...
```

Reply:
This will be worked on in #127